src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -85,7 +85,7 @@
     <PackageReference Include="System.ServiceModel.Primitives" Version="4.3.0" />
     <PackageReference Include="System.ServiceModel.Security" Version="4.3.0" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.3.0" />
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.3.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.0.11" />
```

Comment:
Why change it to `4.0.11`? I don't even find such a version on [nuget.org](https://www.nuget.org/packages/System.Text.Encoding.CodePages/)